tests/netperf_test.py

```diff
@@ -13,35 +13,40 @@
 import time
 import os
 from devices import board, wan, lan, wlan, prompt
-
 def install_netperf(device):
     # Check version
     device.sendline('\nnetperf -V')
     try:
-        device.expect('Netperf version 2.4', timeout=10)
+        device.expect('Netperf version 2.6', timeout=10)
         device.expect(device.prompt)
     except:
         # Install Netperf
         device.sendline('apt-get update')
-        device.expect(device.prompt, timeout=60)
-        device.sendline('apt-get -o DPkg::Options::="--force-confnew" -y --force-yes install netperf')
-        device.expect(device.prompt, timeout=60)
-    device.sendline('/etc/init.d/netperf restart')
-    device.expect('Restarting')
-    device.expect(device.prompt)
+        device.expect(device.prompt, timeout=120)
+        device.sendline('apt-get  install netperf')
+        device.expect(device.prompt, timeout=120)
```

Comment:
I think all these lines should be in the except: stanza... maybe the space vs. tabs is showing on the github web page here?